spec/models/new_registration_spec.rb

```diff
@@ -166,11 +166,15 @@
         end
       end
 
-      context "when the address is not a site address" do
-        let(:address) { matching_registration.transient_addresses.first }
+      context "when the address is a contact address" do
```

Comment:
I think these three "when the address is a X address" tests may need some differences in their contexts – as it is, it just looks like the same test three times